src/test/java/com/google/step/datamanager/TagManagerDatastoreTest.java

```diff
@@ -0,0 +1,64 @@
+package com.google.step.datamanager;
+
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.step.model.Tag;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class TagManagerDatastoreTest {
+
+  private static final String NAME_A = "1for1";
+  private static final String NAME_B = "tea";
+
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(
+          new LocalDatastoreServiceTestConfig().setApplyAllHighRepJobPolicy());
+
+  private final TagManagerDatastore tagManagerDatastore = new TagManagerDatastore();
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  @Test
+  public void testGetTag_newTag() {
+    Tag tag = tagManagerDatastore.readOrCreateTagByName(NAME_A);
+    Assert.assertEquals(NAME_A, tag.name);
```

Comment:
Similar comment as the other PR, you can just do a static import for `assertEquals` and get rid of the class qualifier everywhere.